consensus/istanbul/backend/snapshot.go

```diff
@@ -179,12 +179,13 @@ func (s *Snapshot) apply(headers []*types.Header, gov governance.Engine, addr co
 				}
 				gov.ClearVotes(number)
 			}
-			// Reload governance values because epoch changed
-			snap.Epoch, snap.Policy, snap.CommitteeSize = getGovernanceValue(gov, number)
 			snap.Votes = make([]governance.GovernanceVote, 0)
 			snap.Tally = make([]governance.GovernanceTallyItem, 0)
 		}
 
+		// Reload governance values
```

Comment:
According to Ian, the committee size can be changed at any block numbers as contract based GovParam can change the committee size at anytime.

Reply:
Thanks. I'll add it in the description.